policy/modules/system/systemd.fc

```diff
@@ -61,6 +61,11 @@ HOME_DIR/\.config/containers/systemd(/.*)?		gen_context(system_u:object_r:system
 HOME_DIR/\.config/systemd(/.*)?		gen_context(system_u:object_r:systemd_conf_home_t,s0)
 HOME_DIR/\.local/share/systemd(/.*)?		gen_context(system_u:object_r:systemd_data_home_t,s0)
 
+# homed files
+HOME_DIR/\.identity	--	gen_context(system_u:object_r:systemd_homed_record_t,s0)
+HOME_DIR/\.identity-blob(/.*)?	gen_context(system_u:object_r:systemd_homed_record_t,s0)
```

Comment:
Are these intended/required to be inaccessible to the user?

Reply:
I did actually consider that. Looking at accountsservice and the various other things that are working with homed, I think the preferred way of getting account info is from the system userdb stuff. It has sensitive data (like the hashed password for that user) which is why I'm hesitant to make it readable by a user domain, but it is just for their own user. However, I can't see any reason for that information to be read or written to by the user, infact the records are signed so a user cannot write to it anyway. I'm happy to make it readable by the user if you wish however

Reply:
No, that's fine. I'm simply making sure I understand the intent.